packages/zowe-explorer/__tests__/__unit__/command/tsoCommandHandler.unit.test.ts

```diff
@@ -12,14 +12,14 @@
 jest.mock("Session");
```

Comment:
Same as previous comment, but feel free to ignore for now. 😋 
we can always do this at a later time when we address the SonarCloud items you mentioned.

>Sonarcloud failing comes from existing tests just being modified for this PR, maybe in the future we can look into also simplifying our codebase on tests to allow new contributors (or existing) to modify/create tests in a more intuitive way.